std/assembly/util/error.ts

```diff
@@ -5,6 +5,10 @@
 @lazy @inline
 export const E_INDEXOUTOFRANGE: string = "Index out of range";
 
+// @ts-ignore: decorator
+@lazy @inline
+export const E_ATOMICINVALIDINDEX: string = "Invalid atomic access index";
```

Comment:
v8/node expect [special error range](https://github.com/ajamshed/node.js/blob/9faeacbd97a09024ee984a0140f7dac2af74bc20/deps/v8/src/js/harmony-atomics.js#L46) for this case

Reply:
I see, thanks. The [spec](https://tc39.es/ecmascript_sharedmem/shmem.html#Atomics.ValidateAtomicAccess) states `If accessIndex < 0 or accessIndex ≥ length, throw a RangeError exception.`, not making any other restrictions. Reason I'm asking is because it might be preferable to keep static error message data minimal, and reusing can help with that.

Reply:
Ok, I'll reuse `E_INDEXOUTOFRANGE` constant